ysoserial/Generators/WindowsPrincipalGenerator.cs

```diff
@@ -123,16 +125,9 @@ public override object Generate(string formatter, InputArgs inputArgs)
     </WindowsPrincipal>
 </root>";
 
-                if (inputArgs.Minify)
-                {
-                    if (inputArgs.UseSimpleType)
-                    {
-                        payload = XMLMinifier.Minify(payload, new string[] { "mscorlib" }, null);
-                    }
-                    else
-                    {
-                        payload = XMLMinifier.Minify(payload, null, null);
-                    }
+                // just remove spaces, not xmlns attribute values, also no xxe here kids
+                if (inputArgs.Minify) {
```

Comment:
Perhaps add the minifier again to make sure it is being minified correctly when using full type in the payloads